app/questionnaire/rules.py

```diff
@@ -257,5 +275,10 @@ def get_metadata_value(metadata, key):
     return metadata.get(key)
 
 
+def get_list_count(list_store, list_name):
+    list_values = list_store[list_name]
+    return len(list_values) if list_values else 0
```

Comment:
Since you always get an empty list back?
```suggestion
    return len(list_values)
```

Reply:
I've also removed `list_values` so this is now `return len(list_store[list_name])`.